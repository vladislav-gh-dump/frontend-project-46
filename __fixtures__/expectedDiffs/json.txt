[
  {
    "stat": "nested",
    "key": "common",
    "children": [
      {
        "stat": "received",
        "key": "follow",
        "value": false
      },
      {
        "stat": "matched",
        "key": "setting1",
        "value": "Value 1"
      },
      {
        "stat": "expected",
        "key": "setting2",
        "value": 200
      },
      {
        "stat": "exchanged",
        "key": "setting3",
        "values": {
          "from": true,
          "to": null
        }
      },
      {
        "stat": "received",
        "key": "setting4",
        "value": "blah blah"
      },
      {
        "stat": "received",
        "key": "setting5",
        "value": {
          "key5": "value5"
        }
      },
      {
        "stat": "nested",
        "key": "setting6",
        "children": [
          {
            "stat": "nested",
            "key": "doge",
            "children": [
              {
                "stat": "exchanged",
                "key": "wow",
                "values": {
                  "from": "",
                  "to": "so much"
                }
              }
            ]
          },
          {
            "stat": "matched",
            "key": "key",
            "value": "value"
          },
          {
            "stat": "received",
            "key": "ops",
            "value": "vops"
          }
        ]
      }
    ]
  },
  {
    "stat": "nested",
    "key": "group1",
    "children": [
      {
        "stat": "exchanged",
        "key": "baz",
        "values": {
          "from": "bas",
          "to": "bars"
        }
      },
      {
        "stat": "matched",
        "key": "foo",
        "value": "bar"
      },
      {
        "stat": "exchanged",
        "key": "nest",
        "values": {
          "from": {
            "key": "value"
          },
          "to": "str"
        }
      }
    ]
  },
  {
    "stat": "expected",
    "key": "group2",
    "value": {
      "abc": 12345,
      "deep": {
        "id": 45
      }
    }
  },
  {
    "stat": "received",
    "key": "group3",
    "value": {
      "deep": {
        "id": {
          "number": 45
        }
      },
      "fee": 100500
    }
  }
]