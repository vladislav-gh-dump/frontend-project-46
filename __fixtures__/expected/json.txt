[
  {
    "stat": "nested",
    "key": "common",
    "value": [
      {
        "stat": "received",
        "key": "follow",
        "value": false
      },
      {
        "stat": "matched",
        "key": "setting1",
        "value": "Value 1"
      },
      {
        "stat": "expected",
        "key": "setting2",
        "value": 200
      },
      {
        "stat": "exchanged",
        "key": "setting3",
        "value": [
          true,
          null
        ]
      },
      {
        "stat": "received",
        "key": "setting4",
        "value": "blah blah"
      },
      {
        "stat": "received",
        "key": "setting5",
        "value": {
          "key5": "value5"
        }
      },
      {
        "stat": "nested",
        "key": "setting6",
        "value": [
          {
            "stat": "nested",
            "key": "doge",
            "value": [
              {
                "stat": "exchanged",
                "key": "wow",
                "value": [
                  "",
                  "so much"
                ]
              }
            ]
          },
          {
            "stat": "matched",
            "key": "key",
            "value": "value"
          },
          {
            "stat": "received",
            "key": "ops",
            "value": "vops"
          }
        ]
      }
    ]
  },
  {
    "stat": "nested",
    "key": "group1",
    "value": [
      {
        "stat": "exchanged",
        "key": "baz",
        "value": [
          "bas",
          "bars"
        ]
      },
      {
        "stat": "matched",
        "key": "foo",
        "value": "bar"
      },
      {
        "stat": "exchanged",
        "key": "nest",
        "value": [
          {
            "key": "value"
          },
          "str"
        ]
      }
    ]
  },
  {
    "stat": "expected",
    "key": "group2",
    "value": {
      "abc": 12345,
      "deep": {
        "id": 45
      }
    }
  },
  {
    "stat": "received",
    "key": "group3",
    "value": {
      "deep": {
        "id": {
          "number": 45
        }
      },
      "fee": 100500
    }
  }
]